src/dodal/beamlines/b01_1.py

```diff
@@ -0,0 +1,77 @@
+from pathlib import Path
+
+from ophyd_async.epics.adaravis import AravisDetector
+from ophyd_async.fastcs.panda import HDFPanda
+
+from dodal.common.beamlines.beamline_utils import (
+    device_instantiation,
+    get_path_provider,
+    set_path_provider,
+)
+from dodal.common.beamlines.beamline_utils import set_beamline as set_utils_beamline
+from dodal.common.visit import LocalDirectoryServiceClient, StaticVisitPathProvider
+from dodal.devices.synchrotron import Synchrotron
+from dodal.log import set_beamline as set_log_beamline
+
+BL = "c01"
+set_log_beamline(BL)
+set_utils_beamline(BL)
+
+set_path_provider(
+    StaticVisitPathProvider(
+        BL,
+        Path("/dls/b01-1/data/"),
+        client=LocalDirectoryServiceClient(),
+    )
+)
+
+"""
+NOTE: Due to ArgoCD and the k8s cluster configuration those PVs are not available remotely.
+You need to be on the beamline-local network to access them.
+The simplest way to do this is to `ssh b01-1-ws001` and run `dodal connect b01_1` from there.
+remember about the underscore in the beamline name.
+
+See the IOC status here:
+https://argocd.diamond.ac.uk/applications?showFavorites=false&proj=&sync=&autoSync=&health=&namespace=&cluster=&labels=
```

Comment:
Should: This link doesn't work for me, I just see an empty screen in argo. Do I need specific permissions?

Reply:
I don't think so, it just worked for me when I log in with CAS.

Maybe I was added to the allowlist though

Reply:
yes, it's the same as permissions for the cluster

Reply:
Ok, maybe add a note to that affect then